src/v/raft/group_manager.h

```diff
@@ -109,7 +109,7 @@ class group_manager {
     recovery_throttle& _recovery_throttle;
     recovery_memory_quota _recovery_mem_quota;
     features::feature_table& _feature_table;
-    bool _is_ready;
+    bool _is_ready{false};
```

Comment:
since there is no defaulted constructor, would it be preferable to initialize in the existing constructor?

Reply:
I'm indifferent. Changed.